pages/dev/ui-kit/index.tsx

```diff
@@ -44,24 +55,55 @@ const Login = () => {
             </div>
           </div>
         </div>
+        <div className={style.kitBlock}>
+          <h2 className={style.componentHeader}>Textarea</h2>
+          <div className={style.components}>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <TextArea color={TEXTAEREA_COLORS.DEFAULT} />
+            </form>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <TextArea color={TEXTAEREA_COLORS.ACTIVE} isActive={true} />
+            </form>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <TextArea color={TEXTAEREA_COLORS.ERROR} hasError={true} />
+            </form>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <TextArea color={TEXTAEREA_COLORS.HOVER} isHovered={true} />
```

Comment:
you need to throw this logic inside, classes like hover, active should be used in the component automatically, you don't need to write them manually.